eks-anywhere-common/Addons/Partner/SUSE/NeuVector/neuvector-crd.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
You are having two HelmReleases with same name, the installation fails because of that. Also consider using `kustomization.yaml`.